topbin/coqnative_bin.ml

```diff
@@ -236,12 +236,9 @@ let init_load_path includes =
   let user_contrib = coqlib/"user-contrib" in
   let xdg_dirs = Envars.xdg_dirs in
   let coqpath = Envars.coqpath in
-  let plugins = coqlib/"plugins" in
   (* NOTE: These directories are searched from last to first *)
   (* first standard library *)
   add_rec_path ~unix_path:(coqlib/"theories") ~coq_root:(Names.DirPath.make[coq_root]);
-  (* then plugins *)
-  add_rec_path ~unix_path:plugins ~coq_root:(Names.DirPath.make [coq_root]);
```

Comment:
I see, this code got outdated due to the other PRs (in concrete the one doing the build layout) ; you can still keep the `add_rec_path` if you update the above line to be `coqcorelib/"plugins"`.

Reply:
The native compiler only needs access to the `vo` files, so there is little point in adding the `plugins` directory to the path.